### PR TITLE
fix: clear cache on logout

### PR DESCRIPTION
**Pull request by ysael**
clear cache from react-query on logout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -67,27 +66,23 @@ export default function App() {
                   {/* Page Content */}
                   <ScrollToTop />
                   {/* Scroll to top on page change */}
-                  <SubscriptionRouteManager stripePromise={stripePromise} signOut={signOut}>
+                  <SubscriptionRouteManager
+                    stripePromise={stripePromise}
+                    signOut={() => {
```
Comment (lawyu89) [inline src/App.js:71]: you'll need to apply the same logic to line 89 since it passes signOut there too.

Comment (lawyu89) [reply src/App.js:71]: gonna pre approve so youre not blocked

**Review by lawyu89 — APPROVED** [review]
LGTM, just need address comment

File: src/App.js (hunk 1 above)
Comment (lvancraen) [reply src/App.js:71]: I think we should create a wrapper for the signOut function. I'm actually looking into a bug due to jotai causing the state of the atoms to remain after logout. Without refreshing the page, the state of a previous user will show in the next user (this is the bug people have been facing with the care plans showing up even when it's a new user). I'm creating a PR based off of this one to fix it.

Comment (lvancraen) [reply src/App.js:71]: PR https://github.com/capable-health/capable-care-demo/pull/66

Comment (ysael) [reply src/App.js]: thanks @lvancraen and @lawyu89 I will test this a bit more and merge if all issues are gone 🥳

**Pull request merged**